Formula/dmd.rb

```diff
@@ -31,30 +31,25 @@ class Dmd < Formula
   end
 
   head do
-    url "https://github.com/dlang/dmd.git"
+    url "https://github.com/dlang/dmd.git", branch: "master"
 
     resource "druntime" do
-      url "https://github.com/dlang/druntime.git"
+      url "https://github.com/dlang/druntime.git", branch: "master"
     end
 
     resource "phobos" do
-      url "https://github.com/dlang/phobos.git"
+      url "https://github.com/dlang/phobos.git", branch: "master"
     end
 
     resource "tools" do
-      url "https://github.com/dlang/tools.git"
+      url "https://github.com/dlang/tools.git", branch: "master"
     end
   end
 
+  depends_on "ldc" => :build
   depends_on arch: :x86_64
```

Comment:
Do we still need this now that we have something we can bootstrap with on ARM?

Reply:
My hunch is that `dmd` itself doesn't work on ARM but since I'm making additional changes here anyways, I'll remove this and see what happens.